packages/database/src/migrations/20210305010124-AddMissingCasesToSyncQueue-modifies-data.js

```diff
@@ -29,18 +29,51 @@ const createSelectCasesQuery = columns => {
 };
 
 const selectCasesForResync = async db => {
-  const query = createSelectCasesQuery('*');
+  const query = createSelectCasesQuery();
```

Comment:
Kostas' version used a separate function to create the query because he was reusing parts of the query in multiple places. In your PR, you're using each query just once, so just put it here rather than calling a function to create it.

FYI if you were using it multiple places, but the function took no parameters (as it does now) and immediately returned the string, you'd declare that string as a `const` instead.